### DISCUSSION TITLE
I lost my github account

### DISCUSSION THREAD

**Discussion by legentsking** [community/community#39841] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can't login to my github account because i changed my mobile and forgot to take 2F Authenticator app backup and don't have backup codes also and also don't have SSH key to verify the last used device. 
I only have is the gmail id.

Please help me to recover that account.

How can i get that account back .

**Comment by david-engelmann** [comment]
[Check out this guide on recovering an account if you lose your 2FA access](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)!

**Comment by DSDmark** [comment]
Hi @legentsking,

try above solution. If you have nothing for doing in that matter then last option to contact with GitHub-support https://support.github.com/request
Go there and write mail to GitHub support.
